static/js/publisher/release/actions/pendingReleases.js

```diff
@@ -37,8 +65,19 @@ export function promoteRevision(revision, channel) {
       );
     });
 
+    const canBeProgressive = revision.architectures.some(arch =>
+      hasRelease(getState(), channel, arch)
+    );
+
     if (!isAlreadyReleased) {
-      dispatch(releaseRevision(revision, channel));
+      dispatch(
+        releaseRevision(
+          revision,
+          channel,
+          undefined,
+          canBeProgressive ? true : undefined
```

Comment:
Is it used anywhere? It seems that this function takes 3 params, not 4: `export function releaseRevision(revision, channel, progressive) {`

Reply:
Ah thanks, missed this one... I originally added this, but then realised I could just do it in one place